File: latihanB/java/Ram.java
public class Ram {
    private int capacity;
    private int price;

    Ram() {}

    Ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getCapacity() {
        return this.capacity;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    public void printRam() {
        System.out.println("Capacity\t: " + this.capacity + " GB");
        System.out.println("Price\t\t: " + this.price + " IDR");
    }
}

File: latihanB/php/Ram.php
<?php 

class Ram {
    private $capacity,
            $price;

    function __construct($capacity, $price) {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }

    function getCapacity() {
        return $this->capacity;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    function printRam() {
        echo "Capacity\t: " . $this->capacity . " GB<br>";
        echo "Price\t\t: " . $this->price . " IDR<br>";
    }

    function __destruct() { }
}

?>